modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -349,4 +349,11 @@ export class Transaction extends BaseTransaction {
       memo: txJson.memo,
     };
   }
+
+  /**
+   * Check if this transaction is a P chain
+   */
+  get isTransactionForPChain(): boolean {
```

Comment:
I think import to P should be the default. Could we replace this function with one called `isImportToC`?